### PR TITLE
[Explorer]: update toast

### PR DESCRIPTION
**Pull request by plam-ml**
## Description 

https://mysten.atlassian.net/browse/APPS-1266
message

## Test Plan 

How did you test the new or updated feature?

---
If your changes are not user-facing and not a breaking change, you can skip the following section. Otherwise, please indicate what changed, and then add to the Release Notes section as highlighted during the release process.

### Type of Change (Check all that apply)

- [ ] protocol change
- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

File: apps/explorer/src/components/Layout/index.tsx [hunk 1]
```diff
@@ -95,18 +95,22 @@ export function Layout() {
...
+                                <Banner
+                                    shadow
+                                    border
+                                    variant={
+                                        toast.type === 'success'
+                                            ? 'positive'
+                                            : 'error'
+                                    }
+                                >
+                                    {toast.message as string}
```
Comment (pchrysochoidis) [inline apps/explorer/src/components/Layout/index.tsx:110]: From [the docs of react-hot-toast](https://react-hot-toast.com/docs/toaster) they suggest to use `resolveValue()`

File: apps/explorer/src/components/Layout/index.tsx [hunk 2]
```diff
@@ -95,18 +95,22 @@ export function Layout() {
...
-                                },
                             }}
-                        />
+                        >
+                            {(toast) => (
+                                <Banner
+                                    shadow
+                                    border
+                                    variant={
+                                        toast.type === 'success'
```
[the remark this answers is not in the document]
Comment (plam-ml) [reply apps/explorer/src/components/Layout/index.tsx]: I added the default case for this. Though I believe `toast` function can only do `success` and `error.

[the remark this answers is not in the document]
Comment (pchrysochoidis) [reply apps/explorer/src/components/Layout/index.tsx]: There are a few types [here](https://github.com/timolins/react-hot-toast/blob/main/src/core/types.ts#LL3C25-L3C78) But yeah we use it mostly for success and error

File: apps/explorer/src/components/Layout/index.tsx [hunk 3]
```diff
@@ -95,18 +95,29 @@ export function Layout() {
...
-                                    icon: null,
-                                    className:
-                                        '!bg-issue-light !text-issue-dark',
-                                },
                             }}
-                        />
+                        >
+                            {(toast) => {
+                                let variant: BannerProps['variant'] = undefined;
+                                switch (toast.type) {
```
Comment (pchrysochoidis) [inline apps/explorer/src/components/Layout/index.tsx:102]: Usually defining an ObjectMap `Record<ToastType, BannerType>` or Partial ToastTypes is cleaner and easier to extend. Like
```
obj = {success: 'positive', ...}

const variant = obj[toast.type] || 'neutralWhite';
```

**Pull request merged**